ats/configuration.py

```diff
@@ -26,17 +26,31 @@
 #print my_host
 #print my_hostname
 
-#if SYS_TYPE is None or SYS_TYPE.startswith('linux'):
-#    if my_host.startswith('tt') or my_host.startswith('tr'):
-#        SYS_TYPE = "trinity_knl"
-
-from ats import atsMachines
-MACHINE_DIR = atsMachines.__path__
-#MACHINE_DIR = abspath(os.environ.get('MACHINE_DIR',
-#                             os.path.join(sys.prefix, 'atsMachines')))
-MACHINE_OVERRIDE_DIR = os.environ.get('MACHINE_OVERRIDE_DIR')
-if MACHINE_OVERRIDE_DIR:
-    MACHINE_OVERRIDE_DIR = abspath(MACHINE_OVERRIDE_DIR)
+
+# Set MACHINE_DIR by priority
+#
+# 1) MACHINE_OVERRIDE_DIR env var.
+# 2) MACHINE_DIR env var
+# 3) atsMachines.__path__
+
```

Comment:
Can we add `# HACK: MACHINE_DIR and MACHINE__OVERRIDE_DIR used until discussion with dependent projects` ?

Reply:
Yes